huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```diff
@@ -0,0 +1,530 @@
+				Description: `The CCE cluster ID.`,
+			},
+			"host_group_ids": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				Description: `The log access host group ID list.`,
+			},
+			"tags": common.TagsSchema(),
+			"binary_collect": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: `The binary collection switch. Default is false.`,
+			},
+			"log_split": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: `The log split switch. Default is false.`,
+			},
+			"access_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The log access type.`,
+			},
+			"log_group_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The log group name.`,
+			},
+			"log_stream_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The log stream name.`,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The creation time of the CCE access, in RFC3339 format.`,
+			},
+		},
+	}
+}
+
+func cceAccessConfigDeatilSchema(parent string) *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"path_type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `The type of the CCE access.`,
+			},
+			"windows_log_info": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Elem:        hostAccessConfigWindowsLogInfoSchema(),
+				Optional:    true,
+				Description: `The configuration of Windows event logs.`,
+			},
+			"single_log_format": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Elem:     hostAccessConfigSingleLogFormatSchema(),
+				ExactlyOneOf: []string{
+					fmt.Sprintf("%s.multi_log_format", parent),
+				},
+				Description: `The configuration single-line logs. Each log line is displayed
+ as a single log event.`,
+			},
+			"multi_log_format": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Elem:     hostAccessConfigMultiLogFormatSchema(),
+				Description: `The configuration multi-line logs. Multiple lines of exception log
+ events can be displayed as a single log event. This is helpful when you check logs to locate problems.`,
+			},
+			"paths": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Description: `The collection paths.`,
+			},
+			"black_paths": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Description: `The collection path blacklist.`,
+			},
+			"stdout": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: `The standard output switch. Default is false.`,
```

Comment:
please keep the values of the schema definition and the documentation are same.

Reply:
done